e2e/cypress/integration/account_settings/general/fullname_spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Account Settings > Sidebar > General', () => {
+    before(() => {
+        // # Go to Account Settings as new user
+        cy.loginAsNewUser('user-1');
```

Comment:
Newly created user will never be `user-1`.  This command will create user with random username based on timestamp. This can just be,
```suggestion
        cy.loginAsNewUser();
```

Then at the end of this `before` block, you may do `cy.apiGetMe().as('newuser');` so that in test, you'll be able to get the user info as,
```javascript
cy.get('@newuser').then((user) => {
    ...
});
```